tooling/Microsoft.VisualStudio.Mac.RazorAddin/RazorProjectExtension.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Razor;
+using Microsoft.VisualStudio.Mac.LanguageServices.Razor.ProjectSystem;
+using MonoDevelop.Ide.Composition;
+using MonoDevelop.Ide.TypeSystem;
+using MonoDevelop.Projects;
+
+namespace Microsoft.VisualStudio.Mac.RazorAddin
+{
+    internal class RazorProjectExtension : ProjectExtension
```

Comment:
Consider combining this class with DefaultDotNetProjectHost.

Is there a good reason this has to be in Addin itself?

Reply:
I'd rather leave them separate. My reasoning is I like the separation of the bootsrapping code vs. the code that actually impacts what Razor hosts get created 😄